shinydashboard/lantern/modules/endpointsmodule.R

```diff
@@ -31,6 +32,10 @@ endpointsmodule <- function(
 ) {
   ns <- session$ns
 
+  output$anchorlink <- renderUI({
+    HTML("<p>You may also download endpoint data over time in the JSON format as well as current endpoint data in the CSV format as available here by visiting the <a tabindex=\"0\" id=\"downloads_page_link\" class=\"lantern-url\">Downloads Page</a>.</p>")
```

Comment:
I think we can just say "You may also download endpoint data over time in the JSON format by visiting the Downloads Page" since the CSV file is above, so i don't think its necessary to mention it.